tests/test_rust.py

```diff
@@ -0,0 +1,11 @@
+from synapse.synapse_rust import sum_as_string
+
+from tests import unittest
+
+
+class RustTestCase(unittest.TestCase):
+    """Basic tests to ensure that we can call into Rust code."""
+
+    def test_basic(self):
```

Comment:
Can we add `test_rust` to the list of files typechecked by mypy please?

Reply:
All tests are typechecked by default I think?

Reply:
Oh sorry, I was thinking of `disallow_untyped_defs` which isn't on by default.

But let's drop that for now.